cpp/src/core/sroptcryst.h

```diff
@@ -52,12 +54,14 @@ class srTOptCryst : public srTGenOptElem {
 	//TVector3d m_nv; // horizontal, vertical and longitudinal coordinates of outward normal to crystal surface in the frame of incident beam
 	TVector3d m_tv; // horizontal, vertical and longitudinal coordinates of central tangential vector [m] in the frame of incident beam
 	TVector3d m_sv; // horizontal, vertical and longitudinal coordinates of central saggital vector [m] in the frame of incident beam
+	TVector3d v_nv; //* horizontal, vertical and longitudinal coordinates of outward normal to crystal surface in the frame of incident beam */ //ARF06062019 Introduce Vector3d& v_nv,
 	
-	char m_uc; // crystal use case: 1- Bragg Reflection, 2- Bragg Transmission
-	// Laue cases added: 3- Laue, 4- Laue Transmission ARF08052019
-
-	// TRANSFORMATION MATRIX
-	double m_RXtLab[3][3]; // RXtLab: 3x3 orthogonal matrix that converts components of a 3x1 vector in crystal coordinates to components in lab coordinates. 
+	int m_uc; // crystal use case: 1- Bragg Reflection, 2- Bragg Transmission (Laue cases to be added)
+               // case: 3- Laue, 4- Laue Transmission ARF08052019 change char to int
+	int m_ug; //ARF2305219 This variable define Laue (2) or Bragg (1)
```

Comment:
Why should we add this new member variable? The original intention was to have "m_uc" explicitly describing the crystal use-case.

Reply:
Intention was to separate Bragg and Laue case below, where vectors and amplitudes are calculated, during development. Now that the code is working 'm_ug' is not needed